client/components/panels/submit.js

```diff
@@ -126,13 +126,31 @@ function submit (state) {
   <br>
   <div>
     Fill out the form below and drag your JSON file into the well. Multiple submissions from the same team are fine. Fun team names are also fine. Bonus points for including a link to a code repository!
+	
+  </div>
+  
+  <div>
+    By submitting your algorithm you agree to the following:
+	<ul>
+		<li> If your algorithm is among the top three performing algorithms, you agree to provide a complete description so we can compile a review paper, of which you will be co-author (max 4 persons per team).</li>
+		<li> If you are the winning team, you give us permission to give your contact details to our sponsor, Zeiss AG. They would like to invite the winning team to their research facilities in Oberkochen, Germany. 
+		
+			
   </div>
   <br>
 
   <div style=${style.form}>
     <div>
     <span style=${style.label}>team name</span> <input id='name' style=${style.input}>
     </div>
+	<div>
+    <span style=${style.label}>member names</span> <input id='members' style=${style.input}>
+    </div>
+	
+	<div>
+    <span style=${style.label}>affiliation(s)</span> <input id='affiliation' style=${style.input}>
```

Comment:
I'm all for adding real names of the submitters, but affiliation might not make sense for everyone that we'd like to take part -- e.g. some coder not part of a university or company, so maybe drop that one?